src/JavascriptRenderer.php

```diff
@@ -147,6 +147,8 @@ protected function makeUriRelativeTo($uri, $root)
             return $uris;
         }
 
+	$uri = $uri ?? '';
```

Comment:
This project is using spaces and not tabs :)

```suggestion
        $uri = $uri ?? '';
```

Reply:
Thank you for the correction :-)